test/tools/yulrun.cpp

```diff
@@ -74,7 +75,7 @@ pair<shared_ptr<Block>, shared_ptr<AsmAnalysisInfo>> parse(string const& _source
 	}
 }
 
-void interpret(string const& _source)
+void interpret(string const& _source, bool _inspect, bool _enableExternalCalls)
```

Comment:
What is actually happening if `_enableExternalCalls` is not set?

Reply:
Any external call with simply return with no data